code/game/g_turrets.c

```diff
@@ -565,11 +558,11 @@ void turret_base_think(gentity_t *self)
 
             if (target->takedamage && target->health > 0 && !(target->flags & FL_NOTARGET))
             {
-                if (!target->client && target->team && atoi(target->team) == atoi(self->team))
+                if (!target->client && target->team && atoi(target->team) == team)
```

Comment:
self->team was NULL (I'm guessing) because the turrets were on SP maps and didn't get assigned to either team. This caused a segfault when calling atoi(NULL).